packages/orchestration/src/facade.js

```diff
@@ -54,90 +54,47 @@ const makeLocalChainFacade = localchain => {
+ * @param {object} io
+ * @param {Remote<OrchestrationService>} io.orchestration
+ * @returns {Chain<C>}
  */
-const makeRemoteChainFacade = name => {
-  const chainInfo = {
+const makeRemoteChainFacade = (name, { orchestration }) => {
+  const chainInfo = /** @type {CosmosChainInfo} */ ({
     allegedName: name,
     chainId: 'fixme',
-  };
+    connections: {},
+    icaEnabled: true,
+    icqEnabled: true,
+    pfmEnabled: true,
+    ibcHooksEnabled: true,
+    allowedMessages: [],
+    allowedQueries: [],
+  });
   return {
-    /** @returns {Promise<ChainInfo>} */
-    getChainInfo: async () => anyVal,
+    getChainInfo: async () => chainInfo,
     /** @returns {Promise<OrchestrationAccount<C>>} */
     makeAccount: async () => {
       console.log('makeAccount for', name);
-      // @ts-expect-error fake yet
-      return {
-        delegate(validator, amount) {
-          console.log('delegate got', validator, amount);
-          return Promise.resolve();
-        },
-        deposit(payment) {
-          console.log('deposit got', payment);
-          return Promise.resolve();
-        },
-        getAddress() {
-          return {
-            chainId: chainInfo.chainId,
-            address: 'an address!',
-            addressEncoding: 'bech32',
-          };
-        },
-        getBalance(_denom) {
-          console.error('getBalance not yet implemented');
-          return Promise.resolve({ denom: 'fixme', value: 0n });
-        },
-        getBalances() {
-          throw new Error('not yet implemented');
-        },
-        getDelegations() {
-          console.error('getDelegations not yet implemented');
-          return [];
-        },
-        getRedelegations() {
-          throw new Error('not yet implemented');
-        },
-        getUnbondingDelegations() {
-          throw new Error('not yet implemented');
-        },
-        liquidStake() {
-          console.error('liquidStake not yet implemented');
-          return 0n;
-        },
-        send(toAccount, amount) {
-          console.log('send got', toAccount, amount);
-          return Promise.resolve();
-        },
-        transfer(amount, destination, memo) {
-          console.log('transfer got', amount, destination, memo);
-          return Promise.resolve();
-        },
-        transferSteps(amount, msg) {
-          console.log('transferSteps got', amount, msg);
-          return Promise.resolve();
-        },
-        undelegate(validator, amount) {
-          console.log('undelegate got', validator, amount);
-          return Promise.resolve();
-        },
-        withdraw(amount) {
-          console.log('withdraw got', amount);
-          return Promise.resolve();
-        },
-      };
+
+      // FIXME look up real values
+      const hostConnectionId = 'connection-1';
+      const controllerConnectionId = 'connection-2';
+
+      return E(orchestration).makeAccount(
+        hostConnectionId,
+        controllerConnectionId,
+      );
```

Comment:
This change doesn't look right to me. The returned object should be local.

Reply:
oh, maybe I pushed the wrong version because when I rebased this wasn't here